Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -322,14 +423,14 @@ declared identifier.
 Constructors are methods that initialize class instances. That is, when a new instance of a class is being created, 
 using the `new` object syntax, some constructor of the class is called, perhaps a default anonymous one.
 So, constructor declarations only make sense within classes.
-", Replace("method"));
+", Replace("method")); // TODO - remove whole declaration
```

Comment:
Remove this TODO. I think code actions should not consiste of simply removing the code users took their sweat to write in the first place.

Reply:
Agreed.